samples/pubsub/workerpool/main.go

```diff
@@ -0,0 +1,75 @@
+// Copyright 2018 The Go Cloud Development Kit Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// workerpool is a sample application that simulates messages being
+// published about video uploads and subscribed workers transcoding
+// these videos. In a real application the publish and subscribe
+// sides would normally be in separate executables.
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"math/rand"
+	"time"
+
+	"gocloud.dev/pubsub"
+	"gocloud.dev/pubsub/mempubsub"
+	"gocloud.dev/pubsub/psutil"
+	"golang.org/x/sync/errgroup"
+)
+
+func main() {
+	ctx, _ := context.WithTimeout(context.Background(), time.Duration(10)*time.Second)
```

Comment:
You don't need the `time.Duration` cast for a literal.

Reply:
Done.